### PR TITLE
Feature: RDM-3917 Extending automated front end regression tests for create case filters

### PR DESCRIPTION
**Pull request by adnanakgun**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RDM-3917

### Change description ###

Added feature and made the tests pass.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: test/functional-tests/stepDefinitions/definitionFileSteps.js [hunk 1]
```diff
@@ -36,4 +36,11 @@ defineSupportCode(function ({ Given, When, Then, Before, After }) {
     Data.caseType = 'Conditionals';
   });
 
+  Given(/^I have filled out the create case filters$/, function () {
```
Comment (andrewfolga) [inline test/functional-tests/stepDefinitions/definitionFileSteps.js]: Not sure about the name of this step.

File: test/functional-tests/features/caseFilters.feature [hunk 2]
```diff
@@ -0,0 +1,9 @@
+@caseFilters @functional
```
Comment (noronhaa) [inline test/functional-tests/features/caseFilters.feature]: lets make it pretty with some indentation like the other features

File: test/functional-tests/stepDefinitions/definitionFileSteps.js (hunk 1 above)
Comment (andrewfolga) [reply test/functional-tests/stepDefinitions/definitionFileSteps.js]: Shouldn't we be physically selecting the dropdowns here? Sth like this:

 When(/^I select the 'Case type' drop down option for dynamic filters$/, async function () {
    await searchPage.selectJurisdiction(TestData.jurisdiction);
    await searchPage.selectCaseType(TestData.caseType);
  });

File: test/functional-tests/stepDefinitions/create-caseSteps.js [hunk 3]
```diff
@@ -107,5 +109,23 @@ defineSupportCode(function ({ Given, When, Then, Before, After }) {
...
+
+  When(/^I click the 'Start' button$/, async function () {
+    await createCaseStartPage.clickStartButton();
+  });
+
+  Then(/^I will be navigated to 'Create Case' wizard form page$/, async function () {
+    await browser.getCurrentUrl()
+      .then(function(currentUrl) {
+        expect(currentUrl.indexOf('createCaseSingleFormPage') > -1).to.be.true
+      });
```
Comment (noronhaa) [inline test/functional-tests/stepDefinitions/create-caseSteps.js]: could be slightly more readable in future to use `.includes()`

**Pull request merged**